spk/duplicity/Makefile

```diff
@@ -0,0 +1,34 @@
+SPK_NAME = duplicity
+SPK_VERS = 0.7.03
+SPK_REV = 1
+SPK_ICON = src/duplicity.png
+
+DEPENDS = cross/librsync
+BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel cross/netifaces cross/wrapt cross/$(SPK_NAME)
```

Comment:
- Nit: BUILD_DEPENDS comes before DEPENDS
- Line too long. Please put a line break somewhere in here, so you end up with something like: 
```
BUILD_DEPENDS = cross/python cross/setuptools cross/pip cross/wheel 
BUILD_DEPENDS += cross/boto cross/netifaces cross/wrapt cross/$(SPK_NAME)
```
(`cross/boto` added, because this is where it needs to go anyway ;))